FirebaseStorage/Sources/Storage.swift

```diff
@@ -123,6 +123,12 @@ import FirebaseAuthInterop
     }
   }
 
+  /**
+   * Specify the maximum upload chunk size. Values less than 256K (262144) will be rounded up to 256K. Values
+   * above 256K will be rounded down to the nearest 256K multiple. The default is no maximum.
```

Comment:
the comment says default is no maximum but it seems like it's assigned a default value maximum.

Reply:
The default value is the maximum Int64 value which is billions of gigabytes, so effectively unlimited.